### PR TITLE
[travis + heroku] Added automatic attempt at deployment to heroku upon successful linting of changes on master

### PR DESCRIPTION
**Pull request by Binary-Eater**
Need to rerun the follow because the secure `api_key`  is designed to only work with the `Binary-Eater/attendance` repository (hence why I can safely share it in this PR).

```
travis encrypt $(heroku auth:token) --add deploy.api_key --pro
```

### CODE REVIEW

**Comment by Binary-Eater** [comment]
https://cs411-attendance.herokuapp.com/login

File: README.md [hunk 1]
```diff
@@ -96,11 +97,21 @@ If you need to drop all tables due to a schema change or any other reason, execu
 
 * Clone this repository to your computer
 * Run `npm install` in the newly-cloned directory
-* Run `npm run dev`
+* Run `npm run dev**
```
Comment (JParisFerrer) [inline README.md]: this is broken

**Review by JParisFerrer — COMMENTED** [review]
it works on heroku

File: README.md (hunk 1 above)
Comment (Binary-Eater) [reply README.md]: Ya, thanks....... Not sure how I managed to pull that off.

**Review by JParisFerrer — APPROVED** [review]

**Pull request merged**